pkg/cmd/server/start/master_args.go

```diff
@@ -51,6 +52,9 @@ type MasterArgs struct {
 	StartControllers bool
 	PauseControllers bool
 
+	// DNSDomain is the authoritative domain of the DNS server.
```

Comment:
I can remove it but I think it does make sense to be able to configure it from the command line.